#> cat final_path.msg

pathes[] final_path
